<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ResourceKeyboard</class>
    <widget class="QMainWindow" name="ResourceKeyboard">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>440</width>
                <height>412</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>细节</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                    <number>32</number>
                </property>
                <property name="topMargin">
                    <number>16</number>
                </property>
                <property name="rightMargin">
                    <number>32</number>
                </property>
                <property name="bottomMargin">
                    <number>16</number>
                </property>
                <item>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="cam_id_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>ID : </string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLabel" name="cam_id">
                                <property name="text">
                                    <string>cam_01</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="location_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>区域 :</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLabel" name="location">
                                <property name="text">
                                    <string>区域B, 区域A</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="car_image_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>患者照片 :</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QLabel" name="car_image">
                                <property name="minimumSize">
                                    <size>
                                        <width>250</width>
                                        <height>120</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>加载图片中</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" name="car_color_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>患者性别 :</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QLabel" name="car_color">
                                <property name="text">
                                    <string>Red</string>
                                </property>
                            </widget>
                        </item>
                        <!-- <item row="4" column="0">
                            <widget class="QLabel" name="license_image_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>病灶图像 :</string>
                                </property>
                            </widget>
                        </item> -->
                        <item row="4" column="1">
                            <widget class="QLabel" name="license_image">
                                <property name="minimumSize">
                                    <size>
                                        <width>180</width>
                                        <height>80</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>暂无</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                                </property>
                            </widget>
                        </item>
                        <item row="6" column="0">
                            <widget class="QLabel" name="rule_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>初步诊断 :</string>
                                </property>
                            </widget>
                        </item>
                        <item row="6" column="1">
                            <widget class="QLineEdit" name="rule">
                                <property name="text">
                                    <string>待输入</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="0">
                            <widget class="QLabel" name="license_number_label">
                                <property name="font">
                                    <font>
                                        <weight>75</weight>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>患者学号 :</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="1">
                            <widget class="QLineEdit" name="license_number_lineedit"/>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>98</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="ticket_button">
                                <property name="text">
                                    <string>开诊断单</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="delete_button">
                                <property name="text">
                                    <string>删除</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="close_button">
                                <property name="text">
                                    <string>关闭</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
